src/Okta.Sdk/Generated/IUserType.Generated.cs

```diff
@@ -25,7 +25,7 @@ public partial interface IUserType : IResource
 
         string DisplayName { get; set; }
 
-        string Id { get; set; }
+        string Id { get; }
```

Comment:
This might be problematic, I believe this should not be ready only.

Reply:
Ok, good to know. It changed after I ran the generation process using the latest version. I had the same issue with other entities. I will fix it in the spec.